specreduce/fluxcal.py

```diff
@@ -260,6 +257,7 @@ def standard_sensfunc(self, standard, mode='linear', polydeg=9,
         sensfunc_spec = Spectrum1D(spectral_axis=obj_wave, flux=sensfunc_out)
 
         if display is True:
```

Comment:
should modify or add test to make sure this block gets covered. doesn't look like it has been. an issue can be filed to make sure this gets followed up on.

Reply:
I feel like a display test is going to be very involved, so I opened https://github.com/astropy/specreduce/issues/204 instead, as requested.

Reply:
i've done similar matplotlib tests in CI before and it's not that tricky (at least as far as checking whether the code works and makes a plot. testing if the plot is "correct" is a lot more complex...). definitely outside of the scope of this PR, though, so thanks for the issue.